docs/quickstart.md

```diff
@@ -0,0 +1,765 @@
+                        Path: /hello
+                        Method: get
+                    HelloWorldName:
+                    Type: Api 
+                    Properties:
+                        Path: /hello/{name}
+                        Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+
+A lot has happened here. First, we imported required X-ray SDK classes. 
+`xray_recorder` is a global AWS X-ray recorder class instance that starts/ends segments/sub-segments and sends them to the X-ray daemon.
+To build new sub-segments, we use `xray_recorder.in_subsegment` method as a context manager. 
+Using customized sub-segments, we were able to add visible granular sub-traces into our X-ray, separated by the method name.
+Also, we track lambda cold start by setting global variable outside of a handler. The variable is defined only upon lambda initialization. This information provides an overview of how often the runtime is reused by lambda invoked, which directly impacts perceived lambda performance and latency.
+
+To allow the tracking of our Lambda, we need to set it up in our SAM template and add `Tracing: Active` under lambda `Properties` section.
+!!! Info 
+    Want to know more about context managers and understand the benefits of using them? Follow [article](https://realpython.com/python-with-statement/) from Real Python.
+!!! Info
+    If you want to understand how the Lambda execution environment works and why cold starts can occur, follow [blog series](https://aws.amazon.com/blogs/compute/operating-lambda-performance-optimization-part-1/).
+=== "app.py"
+
+    ```python hl_lines="1 11 13 19 21 27"
+    from aws_lambda_powertools import Logger, Tracer
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+    from aws_lambda_powertools.logging import correlation_paths
+
+    logger = Logger(service="APP")
+    tracer = Tracer()
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    @tracer.capture_method
+    def hello_name(name):
+        tracer.put_annotation("User", name)
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    @tracer.capture_method
+    def hello():
+        tracer.put_annotation("User", "unknown")
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    @tracer.capture_lambda_handler
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+    
+Thanks to the help of powertools tracer we have much cleaner code right now. 
```

Comment:
Thanks to the help of powertools -> With powertools

Reply:
done